crates/graphql/src/main.rs

```diff
@@ -73,7 +81,19 @@ async fn graphql(
     req: web::Json<GraphQLRequest>,
 ) -> Result<HttpResponse, Error> {
     let ctx = AppContext::new(data.clone().into_inner());
+    let start = Local::now();
+
     let resp = req.execute(&data.schema, &ctx).await;
+    let end = Local::now();
+    let duration = end - start;
+    let formatted_duration = duration_hhmmssfff(duration);
+
+    if duration > Duration::milliseconds(5000) {
+        warn!(
+            "long graphql request query={:?}, duration={:?}",
+            req, formatted_duration
+        );
+    }
```

Comment:
Don't want to log every response time?

```
graphq_request query={:?} duration={:?}
```

Reply:
Be cool to base64 encode the query string as well (ideally without variables) so can compare query times between requests and over time.